gnoland/website/pages/HOME.md

```diff
@@ -0,0 +1,69 @@
+# Welcome to **Gno.land**
+
+ * [About Gno.land](/about)
+ * [Blogs](https://test3.gno.land/r/gnoland/blog)
+ * [Install `gnokey`](https://github.com/gnolang/gno/)
+ * [Acquire testnet tokens](https://test3.gno.land/faucet)
+ * [Game of Realms](/game-of-realms) - An open worldwide competition for developers to build the best Gnolang smart-contracts.
+
+# Explore new packages.
+
+* r/gnoland
+  * [/r/gnoland/blog](https://test3.gno.land/r/gnoland/blog)
+  * [/r/gnoland/faucet](https://test3.gno.land/r/gnoland/faucet)
```

Comment:
Hello @jaekwon Some of these links are broken. I am not sure if this is intentional or not. But I thought I'd ask.
The broken links are:

- https://test3.gno.land/r/gnoland/faucet
- https://test3.gno.land/r/system/validators

Reply:
thank you for the feedback.

they are actually broken because of a bug, not because they don't exist.
so, let's keep these links right now and fix the bugs.